securedrop/tests/test_source.py

```diff
@@ -515,14 +516,17 @@ def test_why_journalist_key(source_app):
         assert "Why download the journalist's public key?" in text
 
 
-def test_metadata_route(source_app):
-    with source_app.test_client() as app:
-        resp = app.get(url_for('api.metadata'))
-        assert resp.status_code == 200
-        assert resp.headers.get('Content-Type') == 'application/json'
-        assert json.loads(resp.data.decode('utf-8')).get('sd_version') \
-            == version.__version__
-        assert resp.get('server_os') == 16.04
+def test_metadata_route(source_app, mocker):
```

Comment:
@nightwarrior-xxx Why do we have this `mocker` here?

Reply:
that's from [`pytest-mock`](https://pypi.org/project/pytest-mock/)

Reply:
oh yeah this is doubly mocked...